### DISCUSSION TITLE
Future Workflow of diffcalc PR's

### DISCUSSION THREAD

**Discussion by Rekunan** [ppy/osu#19724] [Q&A]
Once score submission is in a better state (anti-cheat, stable vs lazer leaderboard decision reached, etc) what will be the workflow for diffcalc pr's? Will merges and/or Lazer releases be delayed until reworks are fully deployed? Will pr's be moved to a separate repo?

Asking out of curiosity since I recently wondered what if there's a discrepancy between values shown on osu vs osu-web for example.

**Comment by peppy** [comment]
We're still discussing this internally, but it's on our radar. Probably not ready to make any calls just yet (we have enough to attend to already at the current point in time).
